package.json

```diff
@@ -10,8 +10,8 @@
     "build": "node esbuild/esbuild.config.js --build",
     "start": "node $NODE_OPTIONS dist/server.js | bunyan -o short",
     "start:dev": "concurrently -k -p \"[{name}]\" -n \"ESBuild,Node\" -c \"yellow.bold,cyan.bold\" \"node esbuild/esbuild.config.js --build --watch\" \"node esbuild/esbuild.config.js --dev-server\"",
-    "start-feature": "DOTENV_CONFIG_PATH=feature.env node $NODE_DEBUG_OPTION -r dotenv/config dist/server.js | bunyan -o short",
-    "start-feature:dev": "DOTENV_CONFIG_PATH=feature.env concurrently -k -p \"[{name}]\" -n \"ESBuild,Node\" -c \"yellow.bold,cyan.bold\" \"node esbuild/esbuild.config.js --build --watch\" \"node esbuild/esbuild.config.js --dev-server\"",
+    "start-feature": " node $NODE_DEBUG_OPTION --env-file=feature.env dist/server.js | bunyan -o short",
+    "start-feature:dev": "concurrently -k -p \"[{name}]\" -n \"ESBuild,Node\" -c \"yellow.bold,cyan.bold\" \"node --build --watch\" \"node esbuild/esbuild.config.js --dev-test-server\"",
```

Comment:
this won't work since you removed `esbuild/esbuild.config.js`

Reply:
Thanks @ushkarev I've fixed that now